.github/action_templates/build-and-push/action.yaml

```diff
@@ -20,9 +20,9 @@ runs:
           echo "${{ inputs.docker-auth-token }}" | docker login --username awiciroh --password-stdin     
     - name: Build and tag Docker image
       shell: bash
-      run: docker build -t awiciroh/ciroh-ngen-image:${{ inputs.github-sha }} -t awiciroh/ciroh-ngen-image-${{ inputs.image-platform }}:latest docker/.      
+      run: docker build -t awiciroh/ciroh-ngen-image:${{ inputs.github-sha }} -t awiciroh/ciroh-ngen-image:latest-${{ inputs.image-platform }} docker/.      
```

Comment:
I think this build step is where you want to use `QEMU` and then provide the `--platform` flag to the build so it can build either one...